oak_runtime/src/io.rs

```diff
@@ -84,26 +81,32 @@ pub trait SenderExt<T> {
     /// Encodes and sends a message to the [`Sender::handle`].
     fn send(&self, message: T, runtime: &RuntimeProxy) -> Result<(), OakError>;
 
+    /// Encodes and sends a message to the [`Sender::handle`] using the current Node's privilege.
+    fn send_with_privilege(&self, message: T, runtime: &RuntimeProxy) -> Result<(), OakError>;
```

Comment:
Alternatively perhaps `send_with_downgrade` or something like that? (also at the ABI level)

Especially since we are planning to at some point remove the explicit privilege and use integrity as declassification privilege effectively, so I think downgrade is more accurate.

@aferr do you have a preference either way?

Reply:
Changed to use `with_downgrade` throughout